src/components/clusters/ClusterDetailsMultiRegion/components/IdentityProvidersPage/components/HtpasswdDetails/UploadHTPasswdFileModal.tsx

```
@@ -0,0 +1,162 @@
+import ErrorBox from '~/components/common/ErrorBox';
+import Modal from '~/components/common/Modal/Modal';
+import { closeModal } from '~/components/common/Modal/ModalActions';
+import modals from '~/components/common/Modal/modals';
+import { useImportHtpasswdUsers } from '~/queries/ClusterDetailsQueries/AccessControlTab/UserQueries/useImportHtpasswdUsers';
+import { useGlobalState } from '~/redux/hooks';
+
+import {
+  maskHTPasswdFileContent,
+  parseHTPasswdFile,
+  ParsedHTPasswdUser,
+} from '../ProvidersForms/htpasswdFileParser';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)